src/compute_budget.zig

```diff
@@ -0,0 +1,118 @@
+const std = @import("std");
+const sol = @import("solana-program-sdk");
+const bincode = @import("bincode");
+
+const Account = sol.Account;
+
+const ComputeBudget = @This();
+
+pub const id = sol.PublicKey.comptimeFromBase58("ComputeBudget111111111111111111111111111111");
+
+pub fn requestHeapFrame(allocator: std.mem.Allocator, params: struct { bytes: u32 }) !sol.Instruction {
+    const data = try bincode.writeAlloc(allocator, ComputeBudget.Instruction{
+        .request_heap_frame = .{ .bytes = params.bytes },
+    }, .{});
+    defer allocator.free(data);
+
+    return sol.Instruction.from(.{
+        .program_id = &id,
+        .accounts = &[_]Account.Param{},
+        .data = data,
+    });
+}
+
+pub fn setComputeUnitLimit(allocator: std.mem.Allocator, params: struct { units: u32 }) !sol.Instruction {
```

Comment:
I slightly changed a function declaration and now it accepts `std.mem.Allocator` as a first param. IMO, such an approach simplifies testing.
WDYT?